test/modern_bpf/start_tests.cpp

```diff
@@ -6,6 +6,9 @@
 #include <libpman.h>
 #include <gtest/gtest.h>
 
+/* Dimension of every single ring buffer in these tests is 8 MB. */
+#define RINGBUF_BYTES_DIMENSION 8 * 1024 * 1024
```

Comment:
I think in the future we should find a way to define this just once, not 3 times (driver/main.c, here, scap.h).

Reply:
This could use the scap.h definition and I _think_ it's not critical that the userspace/kernel defaults are the same (though it is nice) so we can leave them split

Reply:
ty